flixel/addons/transition/TransitionFade.hx

```diff
@@ -146,48 +146,48 @@ class TransitionFade extends TransitionEffect
 		}
 	}
 
-	function makeSprite(DirX:Float, DirY:Float):FlxSprite
+	function makeSprite(DirX:Float, DirY:Float, region:FlxRect):FlxSprite
```

Comment:
Um, CI seems to be failing because FlxRect isn't imported?

Reply:
woops! I just copy pasted my code from VS Code into Github web browser editor, forgot that!

Reply:
ok